### PR TITLE
Polish Localization Update

### PR DESCRIPTION
**Pull request by sejbr**

### CODE REVIEW

File: cqui.modinfo [hunk 1]
```diff
@@ -13,6 +13,29 @@
...
+      <en_US>Alpha</en_US>
+      <pl_PL>Alpha</pl_PL>
+    </Text>
+    <Text id="CQUI_STABILITY_BETA">
+      <en_US>Beta</en_US>
+      <pl_PL>Beta</pl_PL>
+    </Text>
+    <Text id="CQUI_STABILITY_RELEASECANDIDATE">
+      <en_US>Release Candidate</en_US>
+      <pl-PL>Kandydat do wydania</pl_PL>
```
Comment (Lctrs) [inline cqui.modinfo]: Opening tag should be corrected from `<pl-PL>` to `<pl_PL>`

**Comment by Lctrs** [comment]
Please resolve conflicts in cqui.modinfo.

**Comment by sejbr** [comment]
@Lctrs fixed

**Comment by chaorace** [comment]
Sorry for taking so long to getting around for reviewing this, I've been moving and consequently internet-free for the last couple days. After a quick review I'll be merging this shortly

**Comment by Lctrs** [comment]
Don't merge this as it will broke `cqui.modinfo` file.

**Comment by chaorace** [comment]
Well, that's what merge commits exist for. No worries, I'll be thoroughly
reviewing the changes before I let anything in!

**Comment by sejbr** [comment]
@Lctrs the modinfo was already merged before this PR I've fixed it in this PR after your remark. Read the commits please, or did I commit the fix incorrectly? Haven't been using the github app much.

**Comment by Lctrs** [comment]
You indeed did the things wrong. I'll explain myself when I find some time.

**Comment by Lctrs** [comment]
So, here is what I think happened based on your commits history.

1. You made some changes to the modinfo file
2. You didn't made your changes against the last version of this file so you merged the last version in your commit instead of rebasing it.
3. Git couldn't merge the changes so it leaved the file in a conflict state (as seen at lines 16, 38, and 72).
4. You thought the changes made by git were intentional and commit/pushed the file in a conflict state, which in turns made Git thinks that thoses changes were intentional and wanted by you, and broke the file in your PR.

Sure @chaorace can merge your PR as is and fix it in another commit. But it would be better if you fix it in it.

**Comment by sejbr** [comment]
@Lctrs I see, sorry. I believe the GUI application doesn't have any rebase, also it didn't explicitly inform me about the conflict so I missed it.
This repo is my first time using git for anything so after some reading I hope I rebased everything correctly.
Cheers.

**Comment by Lctrs** [comment]
No problem @sejbr.

For a Git GUI, I would recommend switching to Gitkraken (https://www.gitkraken.com/). It's by far the best I ever tried. It made me quit the command line for all the git related operations.

**Comment by chaorace** [comment]
In terms of GUIs, I've had no success using GitKraken. I was very pleased to see it in the Git Student package (still have my edu address because tech schools are too lazy to revoke anything!), but all I ever managed to accomplish with it is corrupting all of my local repos. My personal preference is SourceTree, which I've found quite useful for occasionally breaking a bunch of uncommitted changes in the same file into separate commits

**Comment by sejbr** [comment]
God after 2 hours of all-out war with GitKraken I finally managed to squash and rebase the changes I added today for the smartbanners.
Nice app althou lacking in clarity somehow but I'll stick with it for now. Thanks again @Lctrs for the recomendation.

**Pull request merged**